courses/ahybrid/v1.0/AHYBRID050/scripts/create_gke.sh

```diff
@@ -16,69 +16,65 @@
 
 source ./scripts/env.sh
 
-# create the gke cluster
 gcloud config set compute/zone ${C1_ZONE}
 gcloud beta container clusters create ${C1_NAME} \
     --machine-type=n1-standard-4 \
     --num-nodes=4 \
     --workload-pool=${WORKLOAD_POOL} \
     --enable-stackdriver-kubernetes \
     --subnetwork=default \
-    --labels=mesh_id=${MESH_ID} \
-    --release-channel=regular
+    --labels mesh_id=${MESH_ID}
 
 # service account requires additional role bindings
 kubectl create clusterrolebinding [BINDING_NAME] \
     --clusterrole cluster-admin --user [USER]
 
-# create the service account for gke-connect
 gcloud iam service-accounts create connect-sa
 
-# assign GSA the role it needs
 gcloud projects add-iam-policy-binding ${PROJECT_ID} \
  --member="serviceAccount:connect-sa@${PROJECT_ID}.iam.gserviceaccount.com" \
  --role="roles/gkehub.connect"
 
-# download the service account key
 gcloud iam service-accounts keys create connect-sa-key.json \
   --iam-account=connect-sa@${PROJECT_ID}.iam.gserviceaccount.com
 
-# register the cluster
 gcloud container hub memberships register ${C1_NAME}-connect \
    --gke-cluster=${C1_ZONE}/${C1_NAME}  \
    --service-account-key-file=./connect-sa-key.json
 
-# config project for Anthos Service Mesh
 curl --request POST \
   --header "Authorization: Bearer $(gcloud auth print-access-token)" \
   --data '' \
   https://meshconfig.googleapis.com/v1alpha1/projects/${PROJECT_ID}:initialize
 
-# download anthos service mesh software
-curl -LO https://storage.googleapis.com/gke-release/asm/istio-1.6.4-asm.9-linux-amd64.tar.gz
-tar xzf istio-1.6.4-asm.9-linux-amd64.tar.gz
-cd istio-1.6.4-asm.9
+
+curl -LO https://storage.googleapis.com/gke-release/asm/istio-1.6.5-asm.7-linux-amd64.tar.gz
+tar xzf istio-1.6.5-asm.7-linux-amd64.tar.gz
+cd istio-1.6.5-asm.7
 export PATH=$PWD/bin:$PATH
 
-kpt pkg get https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/asm@release-1.6-asm .
+kpt pkg get https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages.git/. asm
```

Comment:
What do you think about referencing where you acquired this procedure?

Reply:
So the line in red is consistent with the current docs. However, I was having problems getting the current 1.6.8 release to successfully put telemetry into GCP. 

So I instead decided to replicate the ASM installation found in the 045 lab rewrite, which uses 1.6.5 (written before the 1.6.8 instructions were published). The 1.6.5 install on 45 works, so I copied it to get 050 working as quickly as possible.

The green line is from the 1.6.5 instructions, which are no longer documented on the public web site.